core/src/models/ExportProcessor.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  Model,
+  Table,
+  Column,
+  CreatedAt,
+  UpdatedAt,
+  AllowNull,
+  Is,
+  BelongsTo,
+  BeforeCreate,
+  BeforeSave,
+  ForeignKey,
+  DataType,
+  Default,
+  HasMany,
+} from "sequelize-typescript";
+import { Op, QueryTypes } from "sequelize";
+import { api, config } from "actionhero";
+import * as uuid from "uuid";
+import { Destination } from "./Destination";
+import { APIData } from "../modules/apiData";
+import { StateMachine } from "../modules/stateMachine";
+import { Export } from "./Export";
+import { ExportProcessorOps } from "../modules/ops/exportProcessor";
+
+const ERROR_LEVELS = ["error", "info"] as const;
+export type ExportProcessorErrorLevel = typeof ERROR_LEVELS[number];
```

Comment:
Now that this is being used in 2 places (here and Export.ts), it's probably best to share these types.  You could move them into `src/modules/errors` perhaps